<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICAxisKeyboard</class>
 <widget class="QWidget" name="ICAxisKeyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="yPluse">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>80</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>48</height>
    </size>
   </property>
   <property name="text">
    <string>Y+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="xMInus">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>125</y>
     <width>80</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>48</height>
    </size>
   </property>
   <property name="text">
    <string>X-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="xPluse">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>125</y>
     <width>80</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>48</height>
    </size>
   </property>
   <property name="text">
    <string>X+</string>
   </property>
  </widget>
  <widget class="ICLineEditWithVirtualNumericKeypad" name="speedEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>108</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="zMinus">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>40</y>
     <width>80</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>48</height>
    </size>
   </property>
   <property name="text">
    <string>Z-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zPluse">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>210</y>
     <width>80</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>48</height>
    </size>
   </property>
   <property name="text">
    <string>Z+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="yMinus">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>80</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>48</height>
    </size>
   </property>
   <property name="text">
    <string>Y-</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICLineEditWithVirtualNumericKeypad</class>
   <extends>QLineEdit</extends>
   <header>iclineeditwithvirtualnumerickeypad.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
